src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs

```diff
@@ -225,7 +225,7 @@ public override Task<Empty> Hello(Empty request, ServerCallContext context)
                     ExecutionStartedEvent executionStartedEvent = new ExecutionStartedEvent(-1, request.Input)
                     {
                         Name = request.Name,
-                        Version = request.Version,
+                        Version = request.Version != null ? request.Version : this.extension.Options.DefaultVersion,
```

Comment:
Just a heads-up that @nytian is also making big refactoring changes to this class, and we should make sure that these changes don't get lost.